composer.json

```diff
@@ -75,15 +75,15 @@
         "tedivm/jshrink": "~1.4.0",
         "tubalmartin/cssmin": "4.1.1",
         "web-token/jwt-framework": "^v2.2.7",
-        "webonyx/graphql-php": "^0.13.8",
+        "webonyx/graphql-php": "14.9.0",
         "wikimedia/less.php": "^3.0.0"
     },
     "require-dev": {
         "allure-framework/allure-phpunit": "~1.4",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
         "friendsofphp/php-cs-fixer": "~3.0.0",
         "lusitanian/oauth": "~0.8.10",
-        "magento/magento-coding-standard": "*",
+        "magento/magento-coding-standard": "dev-develop",
```

Comment:
I'm completely disagree to use branch or non versioned type of dependency.
Also this PR mentioned to update `webonyx/graphql-php`. Is this update really needed there?

Reply:
Hello @Den4ik 
We cannot finish this issue with released version of the magento-coding-standard.
Only `develop` branch of this repo has new changes, Including update `webonyx/graphql-php`.
This is temporary solution.
Also, please don't forgot that `platform-health` is development branch.
So, I think we can apply this changes, because it will be fixed by Magento release team.

Reply:
I hope that Magento core team don't forget update it with release

Reply:
It will be changed after releasing new version of the magento-coding-standard.
I remember about it